consensus/consensus_v2.go

```diff
@@ -618,11 +630,11 @@ func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 // try to catch up if fall behind
 func (consensus *Consensus) tryCatchup() {
 	utils.GetLogInstance().Info("tryCatchup: commit new blocks", "blockNum", consensus.blockNum)
-	if consensus.phase != Commit && consensus.mode.Mode() == Normal {
-		return
-	}
+	//	if consensus.phase != Commit && consensus.mode.Mode() == Normal {
+	//		return
+	//	}
 	currentBlockNum := consensus.blockNum
-	consensus.switchPhase(Announce)
+	consensus.phase = Announce
```

Comment:
If i's a hard override, better to be in the switchPhase() too. phase switch on in one place. like "switchPhase(Announce, override: true)"

Reply:
will add it in another pr with 2f+1 signature checking